src/Analyzers/CSharp/CodeFixes/NamespaceSync/CSharpNamespaceSyncCodeFixProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.NamespaceSync;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.NamespaceSync
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.SyncNamespace), Shared]
+    internal sealed class CSharpNamespaceSyncCodeFixProvider : AbstractNamespaceSyncCodeFixProvider
```

Comment:
Since there's no language specific code, can we just fold this into the "core" provider and make it concrete?

Reply:
This _may_ eventually be used in VB, so it was requested that it be split like this

Reply:
i'm a little worried htat VB support wouldn't ocme with this pr (or immediatley after).  this feels jsut as much a VB feature as a C# one.

Reply:
VB implementation is out of the scope of this PR. We should file an issue and discuss implementation separately.

Reply:
Thought on this more, let's do this: VB and C# for analyzer, C# only fix. VB fix requires a lot more work, but doesn't mean we can't at least enable the diagnostic.